localstack/config.py

```diff
@@ -1168,7 +1162,6 @@ def use_custom_dns():
     "CUSTOM_SSL_CERT_PATH",
     "DEBUG",
     "DEBUG_HANDLER_CHAIN",
-    "DEFAULT_REGION",
```

Comment:
I think we should still forward it to the container such that we can inform the user that it is not used anymore and should be removed from their configuration.

Reply:
Fixed, added a comment too